### PR TITLE
[0.29] Document noexcept in CHANGES.rst

### PR DESCRIPTION
**Pull request by matusvalo**

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -83,6 +83,9 @@ Features added
 * A new argument ``--module-name`` was added to the ``cython`` command to
   provide the (one) exact target module name from the command line.
   Patch by Matthew Brett and h-vetinari.  (Github issue #4906)
+  
+* A new keyword ``noexcept`` was added.
```
Comment (da-woods) [inline CHANGES.rst:87]:
```suggestion
* A new keyword ``noexcept`` was added for forward compatibility with Cython 3.
```

**Comment by da-woods** [comment]
Thanks @matusvalo - I made a small addition, but we should definitely list that in the changes

**Comment by matusvalo** [comment]
Thank you @da-woods for reviewing PR! This is also missing in master branch. Should I create separate PR for master or you will merge 0.29.x branch to master?

**Comment by da-woods** [comment]
0.29.x will get merged into master fair soon (probably when something significant get added) so no need for a PR there.

**Pull request merged**